### PR TITLE
add eth_pendingTransactions

### PR DESCRIPTION
**Pull request by winsvega**
connects to https://github.com/ethereum/webthree-umbrella/issues/305

### CODE REVIEW

**Comment by chriseth** [comment]
That's it? Identical to geth already?

**Comment by winsvega** [comment]
no. not yet confirmed. I can't test it. 

ok. it seems in geth this command returns list of pending transactions outcoming from your local accounts. 

I need someone who could check it - creating a pending transaction from local account and see if this command returns such transaction

File: libweb3jsonrpc/Eth.cpp [hunk 1]
```diff
@@ -134,6 +134,25 @@ string Eth::eth_getStorageAt(string const& _address, string const& _position, st
 	}
 }
 
+string Eth::eth_pendingTransactions()
+{
+	//Return list of transaction that being sent by local accounts
+	Transactions ours;
+	for (Transaction pending : client()->pending())
```
Comment (chriseth) [inline libweb3jsonrpc/Eth.cpp]: No space before `:` and please use `Transaction const&`

File: libweb3jsonrpc/Eth.cpp [hunk 2]
```diff
@@ -134,6 +134,25 @@ string Eth::eth_getStorageAt(string const& _address, string const& _position, st
...
+	{
+		for (Address account : m_ethAccounts.allAccounts())
```
Comment (chriseth) [inline libweb3jsonrpc/Eth.cpp]: Also here: `Address const&` and no space

**Pull request merged**